pages/k8s/upgrading.md

```diff
@@ -60,12 +58,12 @@ As with all upgrades, there is a possibility that there may be unforeseen diffic
 
 You should also make sure:
 
--   The machine from which you will perform the backup has sufficient internet access to retrieve updated software
--   Your cluster is running normally
--   Your Juju client and controller/models are running the latest versions (see the [Juju docs][juju-controller-upgrade])
--   You read the [Upgrade notes][notes] to see if any caveats apply to the versions you are upgrading to/from
--   You read the [Release notes][release-notes] for the version you are upgrading to, which will alert you to any important changes to the operation of your cluster
--   You read the [Upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#deprecation) for details of deprecation notices and API changes for Kubernetes 1.24 which may impact your workloads.
+-   The machine from which you will perform the backup has sufficient internet access to retrieve updated software.
+-   Your cluster is running normally.
+-   Your Juju client and controller/models are running the same, stable version of Juju (see the [Juju docs][juju-controller-upgrade]).
+-   You read the [Upgrade notes][notes] to see if any caveats apply to the versions you are upgrading to/from.
+-   You read the [Release notes][release-notes] for the version you are upgrading to, which will alert you to any important changes to the operation of your cluster.
+-   You read the [Upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md#deprecation) for details of Kubernetes deprecation notices and API changes that may impact your workloads.
```

Comment:
i would have removed the . from this one but :man_shrugging:

Reply:
lol, i never know when to full-stop sentences in bullet points.